packages/common/driver-definitions/src/urlResolver.ts

```diff
@@ -92,6 +94,6 @@ export interface IDriverHeader {
 }
 
 declare module "@fluidframework/core-interfaces" {
-	// eslint-disable-next-line @typescript-eslint/no-empty-interface
+	// eslint-disable-next-line @typescript-eslint/no-empty-interface, jsdoc/require-jsdoc
```

Comment:
>, jsdoc/require-jsdoc [](http://example.com/codeflow?start=66&length=21)

Shouldn't be doced?

Reply:
@kian-thompson @vladsud Do you know how we should document this?

Reply:
I'm not sure what this is. @jatgarg Do you have a clue?

Reply:
Also curious 👀

Reply:
Without traction can we file a work item to follow up (and add comment)? Then, I'd be happy to approve. :)

Reply:
I'd rather not continue to block this PR on this, so if there's no info by later today, I will add a tracking issue, note it in a comment, and get this merged.